docs/community/contribute/localization.md

```diff
@@ -0,0 +1,80 @@
+---
+id: localization
+title: How to translate NEAR Docs
+sidebar_label: Docs Localization
+---
+
+## Introduction
+
+Welcome to the [NEAR DevDocs](https://docs.near.org/) translation project.
+This article is aimed to community members that want to help in the localization of NEAR's documentation.
+If you feel comfortable translating from English into your local language, you're more than welcome to contribute with this project.
+
+### Roles
+
+For simplicity, the translation project is hosted on [Crowdin](https://crowdin.com/project/near-docs) and anyone is free to register and join.
+There are 3 basic roles across the Crowdin project:
+
+1. **Translator:** translates documentation
+2. **Proof Reader:** helps proof reading localized documents
+3. **Manager:** manages memberships and helps with technical issues
+
+
+### How to become a Translator
+
+To become a certified NEAR docs translator, you need to pass a pre-translation test by working on a selected document translation. A proof reader will review your document, and if your translation proves to be high-quality, your application will be accepted and you'll be accepted as a NEAR Certified Translator.
```

Comment:
Do we need a form, forum post or Discord channel for developers to submit their application? so they can provide their Crowdin account, translated posts, NEAR account, etc.

Reply:
I spent some time thinking about it, and I feel that the best stream-lined option is to use Crowdin itself for any user that wants to apply as a translator.
They will need a Crowdin account anyways, and since we are using that platform to follow up other things and manage tasks, I think that it's natural that we concentrate everything there.

Anyone can register, join, and open an "application thread" on the `Discussions` tab. Then the translation teams can follow up, send the tests, review it, and manage the details.